charts/crowdsec/templates/lapi-deployment.yaml

```diff
@@ -29,8 +29,9 @@ spec:
         imagePullPolicy: IfNotPresent
         command: ['sh', '-c', 'wget {{ .Values.lapi.dashboard.assetURL }} && unzip metabase_sqlite.zip -d /metabase-data/']
```

Comment:
Can you please add this change to fix the overwrite and make the initContainer works in case of pod deletion.

Reply:
It will also solve this : https://github.com/crowdsecurity/helm-charts/pull/123

Reply:
Applied